### PR TITLE
Fix transparency in skin icon

### PR DESCRIPTION
**Pull request by lumiscosity**
QPainter has a bug where drawing transparency to a freshly initialized, empty QPixmap causes garbage data to be drawn. This broke the rendering of the skin icon. The fix is simply to fill the QPixmap with empty transparent pixels beforehand.

Signed-off-by: lumiscosity <averyrudelphe@gmail.com>

### CODE REVIEW

File: launcher/SkinUtils.cpp
Comment (TheKodeToad) [inline launcher/SkinUtils.cpp]:
```suggestion
            skin.fill(QColorConstants::Transparent);
```

File: launcher/minecraft/auth/MinecraftAccount.cpp
Comment (TheKodeToad) [inline launcher/minecraft/auth/MinecraftAccount.cpp:133]:
```suggestion
    skin.fill(QColorConstants::Transparent);
```

**Review by TheKodeToad — CHANGES_REQUESTED** [review]
Just a suggestion, if you want to keep it the same please format it

**Comment by lumiscosity** [comment]
sorry, i wasn't aware such a constant existed! it's been added now

**Review by TheKodeToad — APPROVED** [review]

**Comment by TheKodeToad** [comment]
~~You forgot to sign off your commits :P~~
Edit: I don't know what that is about

**Comment by Trial97** [comment]
Maybe its because of some GitHub shenanigans  `maple!` vs `lumiscosity` : 
 in the first commit the signoff is with `maple!` but for the rest is `lumiscosity`

From DCO error message:
image

**Comment by lumiscosity** [comment]
could be; i manually amended the first commit to include the signoff text, and the latter two were accepted with github's "Sign off and commit suggestion" button. sorry, i've never made a pr before

**Review by Scrumplex — APPROVED** [review]

**Comment by Scrumplex** [comment]
> sorry, i've never made a pr before

Don't worry about it :D 

It is quite unintuitive honestly

Thanks for your contribution, either way :D

**Comment by Scrumplex** [comment]
when taking a look at the log, your first commit was authored by `maple!` while the sign off says `lumiscosity`. The latter should also be `maple!`, so ideally amend it again and change `lumiscosity` to `maple!`

**Comment by lumiscosity** [comment]
alright, i asked a friend who's more knowledgeable about this stuff who told me to just wipe the whole thing and recommit it. should be fine now

**Comment by TheKodeToad** [comment]
I think we're waiting for #1486 to be merged before merging any prs targetting develop

**Comment by Scrumplex** [comment]
this should fix the build on the older Qt 5 version used in one of the Linux builds

**Comment by Scrumplex** [comment]
*cough*

**Comment by TheKodeToad** [comment]
oh, I forgot we were using older qt versions than that

**Pull request merged**
